### ISSUE TITLE
Unhandled promise rejection. To suppress this warning, add an error handler to your promise chain with .catch() or a try-catch block around your await expression.

### ISSUE THREAD

**Issue by starryprabin** [ubuntu/gnome-shell-extension-appindicator#304]
I get the following error occasionally

Debian GNU/Linux 11 (bullseye)
GNOME Version: 
GNOME Settings: Gnome 3.38.5
APT:  gnome-shell 3.38.4-1 amd64
AppIndicator/KStatusNotifierItem version 40

Unhandled promise rejection. To suppress this warning, add an error handler to your promise chain with .catch() or a try-catch block around your await expression. Stack trace of the failed promise:#012  _seekStatusNotifierItems/<@/home/prabin/.local/share/gnome-shell/extensions/appindicatorsupport@rgcjonas.gmail.com/statusNotifierWatcher.js:141:1#012  _seekStatusNotifierItems@/home/prabin/.local/share/gnome-shell/extensions/appindicatorsupport@rgcjonas.gmail.com/statusNotifierWatcher.js:141:21#012  async*AppIndicatorsStatusNotifierWatcher@/home/prabin/.local/share/gnome-shell/extensions/appindicatorsupport@rgcjonas.gmail.com/statusNotifierWatcher.js:57:14#012  maybeEnableAfterNameAvailable@/home/prabin/.local/share/gnome-shell/extensions/appindicatorsupport@rgcjonas.gmail.com/extension.js:54:33#012  enable@/home/prabin/.local/share/gnome-shell/extensions/appindicatorsupport@rgcjonas.gmail.com/extension.js:59:5#012  _callExtensionEnable@resource:///org/gnome/shell/ui/extensionSystem.js:167:32#012  _enableAllExtensions/<@resource:///org/gnome/shell/ui/extensionSystem.js:601:22#012  _enableAllExtensions@resource:///org/gnome/shell/ui/extensionSystem.js:600:37#012  _sessionUpdated@resource:///org/gnome/shell/ui/extensionSystem.js:628:18#012  _emit@resource:///org/gnome/gjs/modules/core/_signals.js:133:47#012  _sync@resource:///org/gnome/shell/ui/sessionMode.js:195:14#012  popMode@resource:///org/gnome/shell/ui/sessionMode.js:163:14#012  _continueDeactivate@resource:///org/gnome/shell/ui/screenShield.js:510:30#012  deactivate/<@resource:///org/gnome/shell/ui/screenShield.js:501:44#012  finish@resource:///org/gnome/shell/gdm/authPrompt.js:528:13#012  finish@resource:///org/gnome/shell/ui/unlockDialog.js:870:26#012  deactivate@resource:///org/gnome/shell/ui/screenShield.js:501:26#012  ScreenShield/</<@resource:///org/gnome/shell/ui/screenShield.js:109:57#012  _emit@resource:///org/gnome/gjs/modules/core/_signals.js:133:47#012  _convertToNativeSignal@resource:///org/gnome/gjs/modules/core/overrides/Gio.js:169:19

**Comment by aravindhp** [comment]
I am seeing this too on Fedora 34, Gnome 40.4 amd64, AppIndicator/KStatusNotifierItem version 40
```javascript
Unhandled promise rejection. To suppress this warning, add an error handler to your promise chain with .catch() or a try-catch block around your await expression. Stack trace of the failed promise:
_seekStatusNotifierItems/<@/home/aravindh/.local/share/gnome-shell/extensions/appindicatorsupport@rgcjonas.gmail.com/statusNotifierWatcher.js:141:1
_seekStatusNotifierItems@/home/aravindh/.local/share/gnome-shell/extensions/appindicatorsupport@rgcjonas.gmail.com/statusNotifierWatcher.js:141:21
async*AppIndicatorsStatusNotifierWatcher@/home/aravindh/.local/share/gnome-shell/extensions/appindicatorsupport@rgcjonas.gmail.com/statusNotifierWatcher.js:57:14
maybeEnableAfterNameAvailable@/home/aravindh/.local/share/gnome-shell/extensions/appindicatorsupport@rgcjonas.gmail.com/extension.js:54:33
enable@/home/aravindh/.local/share/gnome-shell/extensions/appindicatorsupport@rgcjonas.gmail.com/extension.js:59:5
_callExtensionEnable@resource:///org/gnome/shell/ui/extensionSystem.js:167:32
loadExtension@resource:///org/gnome/shell/ui/extensionSystem.js:350:26
_loadExtensions/<@resource:///org/gnome/shell/ui/extensionSystem.js:588:18
collectFromDatadirs@resource:///org/gnome/shell/misc/fileUtils.js:27:28
_loadExtensions@resource:///org/gnome/shell/ui/extensionSystem.js:567:19
_enableAllExtensions@resource:///org/gnome/shell/ui/extensionSystem.js:597:18
_sessionUpdated@resource:///org/gnome/shell/ui/extensionSystem.js:628:18
init@resource:///org/gnome/shell/ui/extensionSystem.js:56:14
_initializeUI@resource:///org/gnome/shell/ui/main.js:275:22
start@resource:///org/gnome/shell/ui/main.js:172:5
@<main>:1:47
```

**Comment by 3v1n0** [comment]
This has been closed via commit 500130316c2359e035288ad776233dc20d6167b2

**Comment by puzzle9** [comment]
```
 Unhandled promise rejection. To suppress this warning, add an error handler to your promise chain with .catch() or a try-catch block around your await expression. Stack trace of the failed promise:
                                             refreshPropertyOnProxy@/home/puzzle/.local/share/gnome-shell/extensions/appindicatorsupport@rgcjonas.gmail.com/util.js:41:38
                                             _translateNewSignals/<@/home/puzzle/.local/share/gnome-shell/extensions/appindicatorsupport@rgcjonas.gmail.com/appIndicator.js:206:18
                                             _translateNewSignals@/home/puzzle/.local/share/gnome-shell/extensions/appindicatorsupport@rgcjonas.gmail.com/appIndicator.js:205:50
                                             _onProxySignal/<@/home/puzzle/.local/share/gnome-shell/extensions/appindicatorsupport@rgcjonas.gmail.com/appIndicator.js:222:56
                                             _onProxySignal@/home/puzzle/.local/share/gnome-shell/extensions/appindicatorsupport@rgcjonas.gmail.com/appIndicator.js:222:38

```
